primitives/chain-millau/Cargo.toml

```diff
@@ -8,7 +8,7 @@ license = "GPL-3.0-or-later WITH Classpath-exception-2.0"
 
 [dependencies]
 
-fixed-hash = { version = "0.8.0", default-features = false }
+fixed-hash = { git = "https://github.com/paritytech/parity-common", branch = "master", default-features = false }
```

Comment:
why can't we keep consuming from crates.io?

Reply:
There's a clippy error that we get and that was fixed by this: https://github.com/paritytech/parity-common/commit/d3a9327124a66e52ca1114bb8640c02c18c134b8

And the change wasn't published in the latest release of `fixed-hash`. Also fixed-hash seems to be published quite rarely.

Since the problem is in Millau which is only used for testing I would consume fixed-hash from github from the moment and switch back to crates.io after the fix is published. I can add a TODO related to this.

Reply:
Added a TODO related to this